website/serializer_class.ts

```diff
@@ -0,0 +1,184 @@
+// Copyright 2018 Ryan Dahl <ry@tinyclouds.org>
+// All rights reserved. MIT License.
+
+import * as ts from "typescript";
+import { VISITOR, visit } from "./core";
+import { getModifiers, setFilename } from "./util";
+
+// tslint:disable:only-arrow-functions
+
+VISITOR("ClassDeclaration", function(e, node: ts.ClassDeclaration) {
```

Comment:
I don't think all caps for "VISITOR" is appropriate here. "defineVisitor" would be better.

What's the type of e? If the return type is void please add that. Fix here and elsewhere.

Reply:
I used `VISITOR` rather than anything else because otherwise all my lines became more than 80 cols : /
also tsc can check types for e, see https://github.com/ry/deno/pull/287/files#diff-ce5f89fdedc49f529b701e053aefef25R14